xml/klp.xml

```diff
@@ -111,18 +101,34 @@
    </mediaobject>
   </informalfigure>
 
+  <para>
```

Comment:
On an afterthought, I suggest adding a few words about the diagram explaining the color-coding and the role of the scheduled reboot, so that the reader does not have to decipher it on his/her own.

Reply:
I think that the para that precedes the figure provides an adequate explanation.